### DISCUSSION TITLE
Command line API to mute all feeds that have a particular tag

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6259] [Ideas]
I would like to mute all feeds that have tag as 'youtube', is it possible to do it by executing API request from CLI. Do we have a graphql or rest API for this?

Or is it possible to do it from web interface on browser without having to manually mute feeds one at a time.

**Comment by Alkarex** [comment]
At the moment, an SQL query is the only option to perform such things. But PR welcome
